eng/performance/benchmark_jobs.yml

```diff
@@ -90,8 +103,8 @@ jobs:
             WorkItemTimeout: 4:00 # 4 hours
             ${{ if eq(parameters.osName, 'windows') }}:
               WorkItemDirectory: '$(Build.SourcesDirectory)\docs' # WorkItemDirectory can not be empty, so we send it some docs to keep it happy
-              WorkItemCommand: 'py -3 %HELIX_CORRELATION_PAYLOAD%\scripts\benchmarks_ci.py --incremental no --architecture ${{ parameters.architecture }} -f $(_Framework) --bdn-arguments="$(BenchmarkDotNetArguments)" $(BenchViewArguments)'
+              WorkItemCommand: 'py -3 %HELIX_CORRELATION_PAYLOAD%\scripts\benchmarks_ci.py --incremental no --architecture ${{ parameters.architecture }} -f $(_Framework) --bdn-arguments="$(BenchmarkDotNetArguments)" $(BenchViewArguments) --dotnet-compilation-mode $(_CompilationMode)'
               CorrelationPayloadDirectory: '$(Build.SourcesDirectory)\notLocked' # it gets checked out to a folder with shorter path than WorkItemDirectory so we can avoid file name too long exceptions
             ${{ if ne(parameters.osName, 'windows') }}:
               WorkItemDirectory: '$(Build.SourcesDirectory)'
-              WorkItemCommand: 'python3.5 scripts/benchmarks_ci.py --incremental no --architecture ${{ parameters.architecture }} -f $(_Framework) --bdn-arguments="$(BenchmarkDotNetArguments)" $(BenchViewArguments)'
+              WorkItemCommand: 'python3.5 scripts/benchmarks_ci.py --incremental no --architecture ${{ parameters.architecture }} -f $(_Framework) --bdn-arguments="$(BenchmarkDotNetArguments)" $(BenchViewArguments) --dotnet-compilation-mode $(_CompilationMode)'
```

Comment:
@adamsitnik Shouldn't this be `python3`? In the lab we are (or should be) installing latest Python (3.7.x). This command will work until we stop installing 3.5.

Reply:
I have changed that, I hope it's gonna work. I know it should ;)